tests/openvino/test_modeling.py

```diff
@@ -2179,6 +2179,8 @@ class OVModelForVisualCausalLMIntegrationTest(unittest.TestCase):
         SUPPORT_VIDEO.append("qwen2_5_vl")
     if is_transformers_version(">", "4.49"):
         SUPPORTED_ARCHITECTURES += ["gemma3", "smolvlm"]
+    if is_transformers_version(">", "4.50"):
```

Comment:
shouldn't it be :
```suggestion
    if is_transformers_version(">=", "4.51"):
```
(should not be triggered for transformers v4.50.3 for example)